apps/user/schema.py

```diff
@@ -47,6 +48,7 @@ class Meta:
     display_picture_url = graphene.String()
     organization = graphene.String()
     language = graphene.String()
+    email_display = graphene.String(required=True)
```

Comment:
Change this to display_email.
Right @thenav56 ?

Reply:
`email_display` is fine.